core/src/test/resources/flows/valids/wait-task-flow.yaml

```diff
@@ -0,0 +1,17 @@
+id: wait-task-flow
+namespace: io.kestra.tests
+
+tasks:
+  - id: wait
+    type: io.kestra.plugin.core.debug.Wait
+    delay: PT10S # Waits for 10 seconds before proceeding
+
+  - id: next-task
+    type: io.kestra.plugin.core.scripts.Shell
+    commands:
+      - echo "Wait task completed"
+
+triggers:
```

Comment:
Probably copy past, triggers part is not required, remove it

Reply:
Hii @Skraye 
Thank you for your review and feedback on it. I understand the points you've raised and will make the necessary changes as soon as possible.